### ISSUE TITLE
ListView header scrolling issue

### ISSUE THREAD

**Issue by roberto-ajolfi** [pnp/sp-dev-fx-controls-react#1065]
#### Category
[ ] Enhancement

[ X] Bug

[ ] Question

#### Version

Please specify what version of the library you are using: [ 3.4.1 ]

#### Expected / Desired Behavior / Question
A ListView added in a full page component is limited to a part of the screen and the horizontal / vertical scrolling bars are not working properly.

#### Observed Behavior
A ListView added in a full page component is limited to a part of the screen (instead of scaling to keep all the space available). Because of this, a vertical scroll bar is visible and when I reach the bottom of the list an horizontal scroll bar appears too. When this horizontal scroll bar is visible, the header is no longer scrolling properly. Please check the attached images.

**Initial view with vertical scroll bar**
![listview_proper_scroll_1](https://user-images.githubusercontent.com/4680314/142840786-494e002e-60f2-490f-830e-c9952ead8794.png)

**Initial view with vertical scroll bar - horizontal scrolling works fine (header scrolls too)**
![listview_proper_scroll_2](https://user-images.githubusercontent.com/4680314/142840801-4c50fc47-f6d6-4583-b6ab-4d5cd317fc3c.png)

**Reaching the end of the list - an horizontal scroll bar appears**
![listview_incorrect_scroll_1](https://user-images.githubusercontent.com/4680314/142840829-f64d0f90-6768-43b5-b9a1-015ec4375fc6.png)

**Reaching the end of the list - horizontal scrolling no longer working fine (header do not scroll)**
![listview_incorrect_scroll_2](https://user-images.githubusercontent.com/4680314/142840843-3218d530-0a6c-49fc-ba58-ea5f3cbb2180.png)

#### Steps to Reproduce
- create an SPFX project with a web part (via yeoman)
- add a ListView component (**data loaded via PnP JS**) to the webpart and configure it as a full screen part
- deploy the package, create a new page using the custom part

**Comment by roberto-ajolfi** [comment]
Additional info: it happens when I choose to use the sticky header (`stickyHeader={true}`):
```
<ListView
  items={data}
  viewFields={this.columns}
  compact={false}
  selectionMode={SelectionMode.single}
  selection={this.getSelection}
  stickyHeader={true}
  />
```

**Comment by AJIXuMuK** [comment]
Thank you @roberto-ajolfi for reporting this one.
For the sticky header you need to provide the specific height of the component to make it work.
You can use `className` and `listClassName` properties of the component to specify the height you need. For example, you could do something like `90vh` or `calc(100vh - some_value)`.

Please, get back to us if that doesn't work.

**Comment by roberto-ajolfi** [comment]
Thanks @AJIXuMuK for you reply. Unfortunately the `ListView `doens't seem to have both `className `and `listClassName `property. And it doesn't have a `style `prop too. So I don't know how to apply your suggestion.

**Comment by AJIXuMuK** [comment]
@roberto-ajolfi - it does have both properties: https://github.com/pnp/sp-dev-fx-controls-react/blob/master/src/controls/listView/IListView.ts

Please, check the version of the library if you don't see these props in the List View.

**Comment by roberto-ajolfi** [comment]
@AJIXuMuK I found the props: they're available in v3.5.0. I was using v3.4.1 (as specified in the original request). 
After the upgrade, unfortunately I still experience the same issue.

**Comment by AJIXuMuK** [comment]
Hello @roberto-ajolfi - have you tried to use these properties to play with the styles?
